src/transaction/log_append.cpp

```diff
@@ -1578,6 +1583,9 @@ prior_is_tde_encrypted (const log_prior_node *node)
 static void
 prior_lsa_start_append (THREAD_ENTRY *thread_p, LOG_PRIOR_NODE *node, LOG_TDES *tdes)
 {
+#ifndef CS_MODE
+  assertm(get_server_type() == SERVER_TYPE_TRANSACTION, "Log append can be executed only on transaction server");
+#endif
```

Comment:
- log_append is never used in CS_MODE
- an empty space is required between function/macro name and parenthesis.
```suggestion
  assertm (get_server_type() == SERVER_TYPE_TRANSACTION, "Log append can be executed only on transaction server");
```